Source/Microsoft.Teams.Apps.RemoteSupport/Cards/SmeTicketCard.cs

```diff
@@ -35,21 +35,32 @@ public SmeTicketCard(TicketDetail ticket)
         /// <summary>
         /// Returns an attachment based on the state and information of the ticket.
         /// </summary>
+        /// <param name="cardElementMapping">Represents Adaptive card item element {Id, display name} mapping.</param>
         /// <param name="ticketDetail"> ticket values entered by user.</param>
         /// <param name="applicationBasePath">Represents the Application base URI.</param>
         /// <param name="localizer">The current cultures' string localizer.</param>
         /// <returns>Returns the attachment that will be sent in a message.</returns>
-        public Attachment GetTicketDetailsForSMEChatCard(TicketDetail ticketDetail, string applicationBasePath, IStringLocalizer<Strings> localizer)
+        public Attachment GetTicketDetailsForSMEChatCard(Dictionary<string, string> cardElementMapping, TicketDetail ticketDetail, string applicationBasePath, IStringLocalizer<Strings> localizer)
         {
             ticketDetail = ticketDetail ?? throw new ArgumentNullException(nameof(ticketDetail));
+            cardElementMapping = cardElementMapping ?? throw new ArgumentNullException(nameof(cardElementMapping));
 
             Dictionary<string, string> ticketAdditionalDetail = JsonConvert.DeserializeObject<Dictionary<string, string>>(ticketDetail.AdditionalProperties);
             var dynamicElements = new List<AdaptiveElement>();
             var ticketAdditionalFields = new List<AdaptiveElement>();
 
-            foreach (KeyValuePair<string, string> item in ticketAdditionalDetail)
+            foreach (KeyValuePair<string, string> ticket in ticketAdditionalDetail)
```

Comment:
The items here aren't tickets, they're details about a ticket, so don't call it "ticket"

Reply:
updated to ticketField, as it represents only ticket card field key value pair.